README.md

```diff
@@ -1,11 +1,21 @@
 # commercetools Postman collections
 
-This repository contains Postman collections for the commercetools platform.
+This repository contains Postman collections for the commercetools APIs.
+
+## Why use postman collections?
+
+Postman collections helps you to explore commercetools APIs in a setup time of minutes rather then days or weeks, you can theoratically run the whole commerce journey from creating products to creating, and eventually buying this product for the newly created customer.
```

Comment:
```suggestion
Postman collections help you to explore commercetools APIs in a setup time of minutes rather than days or weeks, you can theoretically run the whole commerce journey from creating products to creating, and eventually buying this product for the newly created customer.
```